CTFd/admin/users.py

```diff
@@ -42,6 +42,20 @@ def users_listing():
                 .order_by(Users.id.asc())
                 .all()
             )
+        elif field == "ip":
+            # Find all the matching user IDs from the tracking table
+            matched_users = (
```

Comment:
Is this more efficient as a subquery somehow? I feel like it would read more easily as `Tracking.query.filter`?

Reply:
`Tracking.query.filter()` returns a list of rows when executed and that's passed directly to `render_template` in the case. This works in the case of other queries because the query is directly run on the users table and no additional querying is required.

I was reading on how to pass a list to an SQLAlchemy query but couldn't find any resources on it. The best solution was to use `subquery` as far as I read but I could be wrong.

Do you have a better solution in mind for this?